### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by acabal** [mgbennet/e-e-smith_first-lensman#1]
- In chapter 7 there are some cases of "the Hill" which are incorrectly cased following a period. se lint catches these.

- For italicized ships that have exclamation points in them, what we should do instead is `<i><span  epub:type="se:name.vessel.ship">Bolivar</span>!</i>`. The exclamation point is not part of the ship's name.

- `<abbr class="initialism">G’s</abbr>` should be `<abbr class="initialism">G</abbr>’s`. The plural is not part of the initialism.

- Metadata: Civilization should be lowercased

- se typogrify makes more changes

- Dedication text should probably be centered

- Endnotes: Name should go in a `<cite>` element, preceded with an em dash within cite

**Comment by acabal** [comment]
Actually, let me revise what I said about the ship names. If the clause is italicized because he's shouting, then the tag to use is `<em>`. However within the clause, the ship name still gets an `<i>` tag. But, because we have italics nested in italics, the ship name should then be set in Roman. For example: 

````
em i{
    font-style: normal;
}

“<em><i epub:type="se:name.vessel.ship">Bolivar</i>! <i epub:type="se:name.vessel.ship">Himalaya</i>!</em>” Clayton barked savagely.
````

Of course that's only the case if it's actually italics for shouted emphasis. I haven't done a close read of it. If not, then the exclamation point still goes outside the italics, because we are only interested in italicizing the ship name and not the complete clause. For example:

````
“<i epub:type="se:name.vessel.ship">Bolivar</i>! <i epub:type="se:name.vessel.ship">Himalaya</i>!” Clayton barked savagely.
````

**Comment by mgbennet** [comment]
That's an interesting point about the shouting and the ship names. I've left out the \<em> as I think it's italicized because it's a ship name and not because he's shouting. There are several other short phrases that are shouted but not italicized in the scans, so I think we're fine to keep it simple.

I've fixed all the other issues except making Civilization lower case in the meta data. Civilization is the word that character use to describe one side of the galactic scale conflict (the other side are referred to as Boskonia or Boskonians). It is capitalized throughout the book. I could maybe reword it to be clearer that the word refers to a collective instead of the concept of civilization.

**Comment by mgbennet** [comment]
Alright, I changed the description to "Virgil Samms, the first individual to recieve a Lens from the mysterious and powerful Arisians, works to create the Galatic Patrol to protect all adherents of Civilization from the threat of the Boskonians and their masters, the Eddorians." I think that makes it a little clearer, but I'm happy to tweak it some more if you want.
